### DISCUSSION TITLE
Minimal installation

### DISCUSSION THREAD

**Discussion by testillano** [ImageMagick/ImageMagick#3186] [General]
Is there any way to install a miminal setup of imagemagick suite in order to save space ?
I only need convert tool inside my docker image, and I don't know if it is possible to reduce the dependencies with any procedure.

thank you !

**Comment by urban-warrior** [comment]
You could simply install the portable version of ImageMagick @ https://download.imagemagick.org/ImageMagick/download/binaries/magick.  Its 14mb and is completely self-contained.  No other files are needed.
